apple/Sources/Sargon/Extensions/SampleValues/Profile+Supporting+Types/References/RefBytes+SampleValues.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+import SargonUniFFI
+
+#if DEBUG
+extension RefBytes {
+	public static var sample: Self { newRefBytesSample() as! Self }
```

Comment:
cannot use `static let sample: Self = newRefBytesSample()` due to covariance.

Reply:
I suppose you could do
```swift
public static let sample: Self = newRefBytesSample() as! Self
```

Reply:
I thought so to, alas, no (feels like a bug actually):

<img width="1443" alt="Screenshot 2024-05-02 at 09 42 59" src="https://github.com/radixdlt/sargon/assets/864410/a23c642e-47c4-4733-92f6-ed1126caeb54">

Reply:
Ah, but what if you just replace `Self` with `RefBytes`?

Reply:
Then it does not fulfill the protocol requirement :D 

```swift
protocol SargonModel {
    static var sample: Self { get }
    static var sampleOther: Self { get }
}
```
oh non final classes... such crap.